rest-api/tests/9-complete-with-all-defence-layers-tests/Tests.System/HealthTests.cs

```diff
@@ -1,22 +1,23 @@
 using System.Net;
 using System.Security.Authentication;
 using Xunit;
+using Xunit.Abstractions;
 
 namespace CompleteWithAllDefenceLayers.Tests.System;
 
-// The Health controller tests verify that we run the correct version, has mandatory JWT access controll,
+// The Health controller tests verify that we run the correct version, has mandatory JWT access control,
 // handle exceptions and return recommended security headers.
 [Trait("Category", "System")]
-public class HealthTests
+public class HealthTests : BaseTests
 {
-    private readonly Uri baseUri = new Uri("https://localhost:5001/");
+    public HealthTests(ITestOutputHelper output) : base(output)
+    {
+    }
 
     [Fact]
     public async Task LivenessAnonymous_ShouldReturn200AndCorrectVersion()
     {
-        var client = new HttpClient();
-
-        var response = await client.GetAsync(new Uri(baseUri, "/api/health/live"));
+        var response = await HttpClient.GetAsync("/api/health/live");
```

Comment:
This should be a call without a token,  var client = new HttpClient();

Reply:
Fixed in amended commit.